core/src/main/scala/kafka/controller/KafkaController.scala

```diff
@@ -348,7 +349,10 @@ class KafkaController(val config: KafkaConfig,
     val childChangeHandlers = Seq(brokerChangeHandler, topicChangeHandler, topicDeletionHandler, logDirEventNotificationHandler,
       isrChangeNotificationHandler)
     childChangeHandlers.foreach(zkClient.registerZNodeChildChangeHandler)
-    val nodeChangeHandlers = Seq(preferredReplicaElectionHandler, partitionReassignmentHandler, topicDeletionFlagHandler)
+
+    zkClient.createLiCombinedControlRequestFlagPath()
```

Comment:
can we make liCombinedControlRequestFlag as a configuration instead of a zk flag? 
Kafka native utils/script can be used to set and retrieve the config.

Reply:
similar to : allowPreferredControllerFallback

Reply:
I've actually added both the config and the zk flag. If both are set, the zk flag will take precedence.
This is done so that in case there are bugs, SREs can dynamically change the zk flag and force a controller move to switch to the previous behavior, without making a config change + a deployment.

Reply:
if the config is added as a dynamic config, then the config can be set at run time without a config change + deployment.

Reply:
Ah, I see what you mean. After some thinking, it seems there are pros in both approaches
1. The implementation in this PR incurs lower overhead as only the preferred controllers are listening on this znode, (and also they are the only ones that would try to create this znode).
2. Adding it as a dynamic broker config, as you did for the allowPreferredControllerFallback, means the current CLI tools are already routing the request via a broker to zk, and don't need to access zk directly. When zk is locked down, this would be a big plus and we don't need to add a separate RPC on brokers just to handle this flag change.

Overall, I feel the 2nd approach as you suggested is probably better since the overhead should be small. I'll update the PR accordingly.